src/server/worker/driver/driver_windows.go

```diff
@@ -12,6 +13,7 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/log"
 	"github.com/pachyderm/pachyderm/v2/src/server/worker/common"
+	"github.com/pachyderm/pachyderm/v2/src/server/worker/logs"
```

Comment:
I guess none of us run on Windows and so goimports didn't add this.  Oops!  We should probably test Windows in CI if people are actually using it.